### ISSUE TITLE
Breadcrumbs JavaScript error on root page listing / index (4.0 RC)

### ISSUE THREAD

**Issue by lb-** [wagtail/wagtail#8992]
### Issue Summary

The breadcrumbs on the page listing under the root page is triggering a JavaScript error.

### Steps to Reproduce

1. Start a new project with `wagtail start myproject`
2. Load up the Wagtail admin
3. Navigate to the root page
4. Expected: No console errors
5. Actual: Console error triggers `Uncaught TypeError: breadcrumbsToggle is null` (note: error is not as clear in the compressed JS build).

-   I have confirmed that this issue can be reproduced as described on a fresh Wagtail project: no

### Technical details

-   Python version: 3.9
-   Django version: 4.0
-   Wagtail version: 4.0 RC1
-   Browser version: Firefox 103.0 (64-bit) on macOS 12.3

**Full error**

```
Uncaught TypeError: breadcrumbsToggle is null
    initCollapsibleBreadcrumbs breadcrumbs.js:50
    <anonymous> wagtailadmin.js:37
    EventListener.handleEvent* wagtailadmin.js:30
    js wagtailadmin.js:72
    __webpack_require__ wagtailadmin.js:184
    __webpack_exports__ wagtailadmin.js:348
    O wagtailadmin.js:221
    <anonymous> wagtailadmin.js:349
    <anonymous> wagtailadmin.js:351
[breadcrumbs.js:50](webpack://wagtail/client/src/includes/breadcrumbs.js?b04c)
    initCollapsibleBreadcrumbs breadcrumbs.js:50
    <anonymous> wagtailadmin.js:37
    (Async: EventListener.handleEvent)
    <anonymous> wagtailadmin.js:30
    js wagtailadmin.js:72
    __webpack_require__ wagtailadmin.js:184
    __webpack_exports__ wagtailadmin.js:348
    O wagtailadmin.js:221
    <anonymous> wagtailadmin.js:349
    <anonymous> wagtailadmin.js:351
```

### Likely root cause

* `data-breadcrumbs-next` is still being rendered even though the root breadcrumbs are not collapsible
* https://github.com/wagtail/wagtail/blob/5ff6922eb517015651b9012bd6534a912a50b449/wagtail/admin/templates/wagtailadmin/pages/page_listing_header.html#L11
* https://github.com/wagtail/wagtail/blob/5ff6922eb517015651b9012bd6534a912a50b449/wagtail/admin/templates/wagtailadmin/shared/breadcrumbs.html#L11
* https://github.com/wagtail/wagtail/blob/5ff6922eb517015651b9012bd6534a912a50b449/client/src/includes/breadcrumbs.js#L14-L16
* Maybe when content is not collapsible we should not be adding the `data-breadcrumbs-next` so that the JS does not try to init the toggle at all OR maybe we should add extra logic to the JS so that it does not try to init the toggle if it is not present in the DOM. Preference would be to ensure the data attribute is not set when JS logic is not required.

### Screenshot

<img width="1557" alt="Screen Shot 2022-08-14 at 3 15 08 pm" src="https://user-images.githubusercontent.com/1396140/184523614-d44f9a69-a2b5-46cc-b638-eeb5e6435f28.png">

**Comment by lb-** [comment]
@PaarthAgarwal  can you please investigate this, confirm it is reproducible, and prepare a fix for this.

Once fixed we will cherry pick the change to the 4.0 branch.

**Comment by PaarthAgarwal** [comment]
> @PaarthAgarwal can you please investigate this, confirm it is reproducible, and prepare a fix for this.
> 
> Once fixed we will cherry pick the change to the 4.0 branch.

Sure✌️

**Comment by mehrdadmoradii** [comment]
I can also reproduce the bug.

### Technical details

- Python version 3.9
- Django version 4.0.7
- Wagtail development version (main branch)
- Web Browser versions:
    - Google Chrome version 103.0.5060.134 (Official Build) (x86_64)
    - Firefox version 103.0.5060.134 (Official Build) (x86_64)

## Steps to reproduce

1. Generate a new project with `wagtail start mysite`.
2. Run the script migration with `python manage.py migrate`, then create a super user with `python manage.py createsuperuser`.
3. Start the development server using `python manage.py runserver`.
4. Navigate the browser to http://127.0.0.1:8000/admin/pages/.

As I followed the above steps the same error has been produced.

The following figures are the screenshots of the error using chrome and Firefox.


<img width="1440" alt="Screen Shot 2022-08-15 at 3 02 17 PM" src="https://user-images.githubusercontent.com/63321262/184702040-4c3695a9-d677-4c3c-802b-e5a0efe98e19.png">


<img width="1440" alt="Screen Shot 2022-08-15 at 3 02 45 PM" src="https://user-images.githubusercontent.com/63321262/184702142-e417dc88-60c2-43f5-99a8-7cdbeb187d03.png">
